mtags/src/main/scala-3/scala/meta/internal/pc/SymbolPrinter.scala

```diff
@@ -23,16 +23,34 @@ class SymbolPrinter(using ctx: Context) extends RefinedPrinter(ctx) {
   }
 
   def fullDefinition(sym: Symbol, tpe: Type): String = {
-    val isImplicit = sym.is(Flags.Implicit)
-    val name = nameString(sym)
-    val implicitKeyword = if (isImplicit) "implicit " else ""
+    def name = nameString(sym)
     keyString(sym) match {
       case "" =>
+        val isImplicit = sym.is(Flags.Implicit)
+        val implicitKeyword = if (isImplicit) "implicit " else ""
         s"$implicitKeyword$name: "
       case key if sym.is(Flags.Method) =>
         s"$key $name"
+      // package
+      case key if sym.is(Flags.Package) =>
+        val owners = for {
+          sym <- sym.ownersIterator
+          name = super.nameString(sym.name)
+          if name != "<root>" && name != "empty"
```

Comment:
```suggestion
          sym <- sym.ownersIterator
           if !(sym.isRoot || sym.isEmptyPackage)
          name = super.nameString(sym.name)
         
```

Reply:
Thanks! I couldn't find a way to do it without string. Fixed